src/configuration/apiConfiguration.ts

```diff
@@ -23,24 +25,23 @@ export interface ApiConfiguration {
 
 export class DefaultApiConfiguration implements ApiConfiguration {
 
-    private _config: any;
+    private _config: RawApiConfiguration;
```

Comment:
Because if it's the result of the xml parsing (without changes).
I tried to find a different name from ApiConfiguration. ApiConfiguration contains a RawApiConfiguration (result of parsing). If you have a better naming...